### DISCUSSION TITLE
modelx gui on spyder instalation on windows

### DISCUSSION THREAD

**Discussion by nedimg071** [fumitoh/modelx#154] [Q&A]
Hi
I tried to get modelx spyder plugins and hoped to get GUI for modelx. It is installed but not recognized by spyder IDE. I tried WPy64-31020-20230513 and GUI for modelx is working normally. Can you give me instructions for adding GUI widgets for modelx please.
Thanks and regards from Nedim

**Comment by fumitoh** [comment]
What's the version of Spyder that you're using? And what is happening? Does Spyder start up normally?

**Reply by nedimg071** [reply]
I had Spyder 5.5.6 installed on windows 11, yesterday changed to Spyder 6.0.0. I installed plugins modelx, spyder_modelx, spyder_kernels, spyder_notebook, spyder_unittest. It is installed on ...AppData\Local\spyder-6\envs\spyder-runtime\Lib\site-packages. Everything is working but I get spyder plugin option on View->Panes for unittest and notebook only. I do not get GUI for modelx. Similar on dependences, I do not get modelx.

**Reply by fumitoh** [reply]
Spyder 6 is not yet supported by spyder-modelx. Is it officially released?

**Reply by nedimg071** [reply]
Yes it was released yesterday :). What about Spyder 5.5.6 and spyder_modelx? Same thing happened there as well. Should I use Spyder 5.2.2 as used on WPy64-31020-20230513?

**Reply by fumitoh** [reply]
Spyder up to version 5.5.3 is supported. 5.5.4+ may work but they're not tested.

**Comment by nedimg071** [comment]
Thank you a lot, it is working now. I used
conda create -n spyder-env -c conda-forge python=3.12
conda activate spyder-env
conda install -c conda-forge spyder=5.5.6
and spyder-modelx can be installed.
